libc/src/__support/complex_type.h

```diff
@@ -62,16 +61,15 @@ template <> struct make_real<cfloat16> {
   using type = float16;
 };
 #endif
-#if defined(LIBC_TYPES_HAS_CFLOAT128) &&                                       \
-    !defined(LIBC_TYPES_CFLOAT128_IS_COMPLEX_LONG_DOUBLE)
+#ifdef LIBC_TYPES_CFLOAT128_IS_NOT_COMPLEX_LONG_DOUBLE
 template <> struct make_real<cfloat128> {
   using type = float128;
 };
 #endif
 
 template <typename T> using make_real_t = typename make_real<T>::type;
 
-template <typename T> LIBC_INLINE constexpr T conjugate(T c) {
+template <typename T> LIBC_INLINE static constexpr T conjugate(T c) {
```

Comment:
If `conjugate` is already in a namespace, does it still need `static` linkage?

Reply:
Why do we need to make conj to be local to the TU?

Reply:
I was just playing around to see if this symbol is exposed in the static archives and object files or not.  Reverted it back to before.

Reply:
I think more generally, we can probably remove this pattern throughout the codebase; I've been noticing that we're not super consistent.